test/integration_test.py

```diff
@@ -311,7 +312,7 @@ def _download_with_drs(self, file_uuid: str):
                 self._validate_fastq_content(file_uuid, content)
                 log.info('Successfully downloaded from %s with DRS', gs_url)
             else:
-                log.warning('Cannot verify content of %s because GOOGLE_APPLICATION_CREDENTIALS is not set.', gs_url)
+                warnings.warn(f'Cannot verify content of {gs_url} because GOOGLE_APPLICATION_CREDENTIALS is not set.')
```

Comment:
Can you run the IT? With only this in place it should fail. That's why I [pointed out](https://github.com/DataBiosphere/azul/pull/1889#discussion_r447309469) #1865 to you.

Reply:
In a phone discussion we decided we should stick with `log.warning` here. I explain why in a comment.